.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@holgiosalos @danielgalvis98 @kliver98 @AlejaGonzalezV @NicolasB2 @manuelq12 @Valeryibarra @veronicatofino
```

Comment:
You forgot *
Check again the step https://github.com/holgiosalos/workshop-api-testing-js-doc#configuraci%C3%B3n-inicial-del-proyecto

Reply:
Also add an enter at the end of the file (EOF)